### PR TITLE
User Story 12

### PR DESCRIPTION
**Pull request by melatran**
User cannot register with an existing email in the system.
Form fields will repopulate with their text but email and password won't.
Error flash messages will still show up.

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -1,16 +1,18 @@
 class UsersController < ApplicationController
 
     def new
-
+      if User.new(user_params).present?
```
Comment (Rostammahabadi) [inline app/controllers/users_controller.rb:4]: This could eventually go into the model as a method called def present?

Comment (Joshua-Tukman) [inline app/controllers/users_controller.rb:4]: Very cool fix! I haven't used this .present? method before.

**Comment by Rostammahabadi** [comment]
Looks great!

File: spec/features/registration/unsuccessful_spec.rb [hunk 2]
```diff
@@ -54,9 +54,9 @@
             address = "123 Main St."
             city = "Denver"
             state = "Colorado"
-            zip = "80209" 
+            zip = "80209"
```
Comment (JPuchalla) [inline spec/features/registration/unsuccessful_spec.rb:57]: Solid Tests.

**Pull request merged**